data/hai/hai ships.txt

```diff
@@ -1433,6 +1593,252 @@ ship "Shield Beetle" "Shield Beetle (Tracker)"
 		`"Bufaer" Atomic Steering`
 		"Hyperdrive"
 
+# low dps but huge ion dmg, use missiles
+ship "Shield Beetle" "Shield Beetle (Ion-ic?)"
+	outfits
+		`"Bondir" Atomic Steering`
+		`"Bondir" Atomic Thruster`
+		"Boulder Reactor"
+		"Geode Reactor"
+		"Hai Corundum Regenerator" 2
+		"Hai Diamond Regenerator" 3
+		"Hai Octopus Jammer" 2
+		"Hai Ravine Batteries"
+		"Hai Williwaw Cooling" 2
+		"Hai Zephyr Cooling" 2
+		Hyperdrive
+		"Ion Cannon" 6
+		"Ionic Blaster Prototype" 2
+		"Outfits Expansion" 6
+		"Pulse Rifle" 55
+
+# heat vulnerability (and low turning, but high speed)
+ship "Shield Beetle" "Shield Beetle (Ionic)"
+	outfits
+		`"Biroo" Atomic Steering`
+		`"Bufaer" Atomic Thruster`
+		"Boulder Reactor"
+		"Geode Reactor"
+		"Hai Diamond Regenerator" 2
+		"Hai Ravine Batteries"
+		"Hai Williwaw Cooling" 2
+		"Hyperdrive"
+		"Ion Cannon" 2
+		"Ionic Blaster Prototype" 4
+		"Ionic Turret Prototype" 2
+		"Pulse Cannon" 2
+		"Pulse Rifle" 95
+		"Hai Octopus Jammer"
+		"Value Detector"
+		"Quantum Keystone"
+		"Supercapacitor" 2
+
+# vulnerability: ion
+ship "Shield Beetle" "Shield Beetle (Sunbeam)"
+	outfits
+		`"Bondir" Atomic Thruster`
+		`"Bufaer" Atomic Steering`
+		"Boulder Reactor"
+		"Sand Cell" 2
+		"Hai Octopus Jammer" 3
+		"Cargo Expansion"
+		"Quantum Keystone"
+		"Bullfrog Anti-Missile" 4
+		"Hai Diamond Regenerator"
+		"Hai Valley Batteries"
+		"Hai Williwaw Cooling" 3
+		"Hyperdrive"
+		"Pulse Rifle" 90
+		"Sunbeam" 8
+
+# slow, waste ammo
+ship "Shield Beetle" "Shield Beetle (Thunderhead)"
+	outfits
+		`"Biroo" Atomic Thruster`
+		`"Bondir" Atomic Steering`
+		"Boulder Reactor"
+		"Hai Williwaw Cooling" 2
+		"Hai Diamond Regenerator" 5
+		"Hai Ravine Batteries"
+		"Hyperdrive"
+		"Outfit Scanner" 4
+		"Pulse Rifle" 80
+		"Pulse Turret" 2
+		"Thunderhead Launcher" 8
+		"Thunderhead Missile" 160
```

Comment:
What about Thunderhead Storage?
```suggestion
		"Thunderhead Storage Array" 8
		"Thunderhead Missile" 160
```

Reply:
the idea is you can waste its ammo without too much difficulty, and I don't think there's space left for that

Reply:
but then it takes ages to deplete the ammo, and this is already one of the strongest variants, with the ammo being limited

Reply:
Okay. In real life, though, I would probably have more ammo. But so that it’s not impossible to fight, we can leave it the way it is. 
Resolved.

Reply:
I'd get more ammo too yeah but the wanderers arent exactly donating it, they give it willingly ~but its difficult to catch them when they explode on you~

Reply:
Totally true

Reply:
also I tried not to spam too many alien weapons on the same ships except if its an actual warlord ship

Reply:
Yes. This can be resolved.